gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/ObservationMapperTest.java

```diff
@@ -118,6 +118,24 @@ public void mapObservationWithValidData() {
         assertThat(observation.hasSubject()).isTrue();
     }
 
+    @Test
```

Comment:
I can see why you've done it this way. But it would make more sense to me if all blood pressures were mapped from the `bloodPreasureMapper`.